engine/engine-core/src/main/scala/org/apache/spark/frame/FrameRdd.scala

```diff
@@ -496,6 +496,26 @@ object FrameRdd {
   }
 
   /**
+   * Converts a spark dataType (as string)to our schema Datatype
+   * @param sparkDataType spark data type
+   * @return a DataType
+   */
+  def sparkDataTypeToSchemaDataType(sparkDataType: String): DataType = {
```

Comment:
Not sure why this is in FrameRDD... seems more likely going in DataTypes or some other more general spark code